src/mangata/mangata.js

```diff
@@ -146,6 +153,10 @@ async function main() {
     const reservedDeltaBN = reservedPlanckDeltaBN.div(liquidityDecimalBN);
     console.log(`${account.name} has compounded ${reservedDeltaBN.toString()} more ${poolName} ...`);
 
+    if (actionSelected !== ScheduleActionType.executeOnTheHour) {
+        return;
+    }
```

Comment:
return is a very rudimentary way to code it. Please refer to src/utils.js and use switch to select code path for different selections.